rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.resolve_index/30_resolve_index_mode_filter.yml

```diff
@@ -0,0 +1,108 @@
+---
+setup:
+  - do:
+      indices.create:
+        index: my-std-index
+        body:
+          settings:
+            number_of_shards: 1
+            number_of_replicas: 0
+          aliases:
+            my-std-alias: { }
+
+  # Create a time-series index
+  - do:
+      indices.create:
+        index: my-ts-index
+        body:
+          settings:
+            index.mode: time_series
```

Comment:
It looks like we support a comma-delimited list, right? It would be good to exercise that somewhere in the yaml rest test.

Reply:
Yes, Right here: https://github.com/elastic/elasticsearch/pull/133616/files/0cbfcd7ee992dbd003b29b5cab85e5f981fe6806#diff-c5c3a90ba94881e41ce06b5a10b8026fbeca951cda3c73515fc8d8e02b3c3138R81

Reply:
Or you mean an index with double mode when created? I have no idea what would happen then tbh...